float64 x_crane
float64 vx_crane
float64 x_load
float64 vx_load
float64 phi_rod
float64 phi_rod_grad
float64 dphi_rod
